src/views/Players/PlayerDetails.js

```diff
@@ -0,0 +1,36 @@
+
+import React, { useEffect, useState } from 'react';
+import { fetchPlayerTeams } from '../../services/fetchPlayers';
+import { useParams } from 'react-router-dom';
+import { Link } from 'react-router-dom';
+
+
+export default function PlayerDetails() {
+  const params = useParams();
+  const id = params.id;
+  const [playerInfo, setPlayerInfo] = useState(null);
+
+  useEffect(() => {
+    const fetchPlayerInfo = async () => {
+      const data = await fetchPlayerTeams(id);
+      setPlayerInfo(data);
+    };
+    fetchPlayerInfo();
+  }, [id]);
+
+  if (!playerInfo) return <h1>...Finding Player Stats</h1>;
```

Comment:
Great handling for the race condition between our state mounting and displaying! Nice job Brenden!